src/app/pages/checkout-payment/payment-payone-ideal-detail/payment-payone-ideal-detail.component.ts

```diff
@@ -0,0 +1,44 @@
+import { ChangeDetectionStrategy, ChangeDetectorRef, Component, Input, OnInit } from '@angular/core';
+
+import { PaymentInstrument } from 'ish-core/models/payment-instrument/payment-instrument.model';
+import { PaymentMethod } from 'ish-core/models/payment-method/payment-method.model';
+
+@Component({
+  selector: 'ish-payment-payone-ideal-detail',
+  templateUrl: './payment-payone-ideal-detail.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class PaymentPayoneIdealDetailComponent implements OnInit {
+  constructor(protected cd: ChangeDetectorRef) {}
+
+  @Input() paymentMethod: PaymentMethod;
+  @Input() paymentInstrument: PaymentInstrument;
```

Comment:
paymentMethod is part of the paymentInstrument. Can the input parameter payment method be omitted?

Reply:
In my opinion paymentMethod contains list of paymentInstruments.